### PR TITLE
Azure Policy extension and assign an Azure Policy for Azure Arc-enabled Kubernetes

### PR DESCRIPTION
**Pull request by lanicolas**
New scenario to integrate Azure Arc-enabled Kubernetes with Azure Policy extension and assign an Azure Policy to the Azure Arc-enabled Kubernetes cluster resource group. By Pablo Ameijeiras

### CODE REVIEW

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/azure_policy/_index.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+  ```shell  
+  . ./azure_policy.sh
+  ```
+
+    > **NOTE: The extra dot is due to the shell script having an _export_ function and needs to have the vars exported in the same shell session as the rest of the commands.**
+
+   The script will:
+
+  - Login to your Azure subscription using the SPN credentials
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/azure_policy/_index.md:125]:
```suggestion
  - Login to your Azure subscription using the service principal credentials
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/azure_policy/_index.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+  - Add or Update your local _connectedk8s_ and _k8s-extension_ Azure CLI extensions
+  - Create Azure Policy cluster extension instance
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/azure_policy/_index.md:127]:
```suggestion
  - Create the Azure Policy cluster extension instance
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/azure_policy/_index.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+        limits:
+          memory: "3Gi"
+          cpu: "16"
+  ```
+
+- When we create the above pod, we get the following output:
+
+  ![Screenshot Azure Policy test](./07.png)
+
+- If there were pods that exceeded the limit before applying the Azure Policy, they will not be removed, but we can see which ones do not comply in the Azure Policy service in the Compliance tab (select the non-compliance event):
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/azure_policy/_index.md:182]:
```suggestion
- If there were pods that exceeded the limit before applying the Azure Policy, they will not be removed, but we can see which ones do not comply in the Azure Policy service in the Compliance tab, select the non-compliance event
```

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/azure_policy/_index.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+
+  ![Screenshot Azure Policy test](./08.png)
+
+- Click on Details to find out which pod is not compliant:
+
+  ![Screenshot Azure Policy test](./09.png)
+
+- This is the non-compliance pod:
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/azure_policy/_index.md:190]:
```suggestion
- This is the non-compliant pod:
```

**Review by sebassem — COMMENTED** [review]
Minor lingo comments
change resource group name to Arc-K8s-Demo to be aligned with other scenarios?

File: docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/azure_policy/_index.md [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+
+    ![Screenshot extension pods on cluster](./05.png)
+
+- Verify under the Azure Policy service in the Assignments tab that the Azure Policy Definition was assigned in your resource group:
+
+  ![Screenshot showing Azure Portal Azure Policy service](./06.png)
+
+## Test Azure Policy
+
+- The Azure Policy we have assigned works as a [LimitRange](https://kubernetes.io/docs/concepts/policy/limit-range/), but we do not specify any namespace, so the Azure Policy will be applied on all namespaces. The limits specified in the Azure Policy are cpuLimit=200m and memoryLimit=1Gi, so to test it we need to create a pod with higher limits:
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/azure_policy/_index.md:158]: Better to show steps on how to apply this deployment to the cluster

**Review by dkirby-ms — CHANGES_REQUESTED** [review]

**Pull request merged**